jerry-core/lit/lit-magic-strings.ini

```diff
@@ -331,6 +331,7 @@ LIT_MAGIC_STRING_TO_DATE_STRING_UL = "toDateString"
 LIT_MAGIC_STRING_TO_TIME_STRING_UL = "toTimeString"
 LIT_MAGIC_STRING_ASYNC_FUNCTION_UL = "AsyncFunction"
 LIT_MAGIC_STRING_ASYNC_ITERATOR = "asyncIterator"
+LIT_MAGIC_STRING_BIGINT64_ARRAY_UL = "BigInt64Array"
```

Comment:
I think this should go above "asyncIterator" because uppercase precede lower-case in ascii charset